Backend/BoardMoveExecutor.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Backend.Data;
+using Backend.Data.Enum;
+using Backend.Data.Struct;
+
+namespace Backend;
+
+public readonly unsafe struct BoardMoveExecutor
+{
+    private readonly struct True { }
+
+    private static readonly delegate*<ref BitBoardMap, void>* FPs;
+
+    [ModuleInitializer]
+    internal static void RunCctor()
```

Comment:
No need to do this. It's already done by `Util.cs`.

Reply:
Please don't tell me you're pre-running all cctors...

Reply:
I am. See https://github.com/TheBlackPlague/StockNemo/blob/4a891767db7cf47449bd06967766d3a1cc2479cf/Backend/Util.cs#L13

Reply:
Does that even work? The ModuleInitializer attribute seems to be missing. Either way, RIP startup perf

Reply:
I believe it works? — Not 100% sure, but running it seems to be fine. 

The start-up performance isn't actually hurt that much.